### ISSUE TITLE
Please, add a configuration to allow the usage of hostile spawners in the area of the big torch! 

### ISSUE THREAD

**Issue by meecosha** [lucaargolo/kibe#127]
Or an in-game modification, like a Soul Big Torch. It's just that mob farms and the Big Torch are mutually exclusive in this case.

**Comment by mapokapo** [comment]
I agree that the big torch should have an option not to influence mob spawners. The Kibe mod page says the big torch is "useful to clean caves" and the reason why most of us light up caves is to increase mob spawns from a mob spawner.
